roles/confluent.zookeeper/defaults/main.yml

```diff
@@ -1,15 +1,15 @@
 ---
 # More zookeeper variables are defined in the confluent.variables role
 
-### Boolean to enable cp-ansible's Custom Log4j Configuration
-zookeeper_custom_log4j: "{{ custom_log4j }}"
+### Boolean to reconfigure Zookeeper's logging with RollingFileAppender and log cleanup
```

Comment:
think on what the customer options should be here

Reply:
update comment to be more clear, but can still keep a var to do the log4j customization, should impact the logic lower down